src/com/serotonin/mango/web/dwr/BaseDwr.java

```diff
@@ -238,7 +240,14 @@ else if (typeId == UserComment.TYPE_POINT)
 
         return c;
     }
-
+    protected String getOwnerNameOfChangePointValue(String pointvalue){
```

Comment:
I assumed the information was displayed in the details of the history of the point so there is no need for an additional method DWR.

Reply:
We do not have information about user name .